app/modules/asset_groups/schemas.py

```diff
@@ -9,6 +9,17 @@
 
 from .models import AssetGroup, AssetGroupSighting
 
+# We need api endpoints for AssetGroupSightings which behave just like the
+# Sighting endpoints to standardize frontend interactions. For that reason we need
+# to know which sighting fields to expect in an AssetGroupSighting.config dict
+SIGHTING_FIELDS_IN_AGS_CONFIG = {
```

Comment:
`SIGHTING_FIELDS_IN_AGS_CONFIG` is only used in tests so I feel like it shouldn't be in `app/` but I don't have a better place to suggest